ci/docs.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - dask-core
   - pip
   - xarray
+  - numpy>=1.20
```

Comment:
@Illviljan i'm bumping minimum numpy here. So PRs to improve the typing using the new numpy types would be very welcome!